### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by mtreinish**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    Python37:
+      python.version: '3.7'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '$(python.version)'
+  displayName: 'Use Python $(python.version)'
+
+- script: |
```
Comment (masayukig) [inline azure-pipelines.yml]: We need to run when the python.version == '2.7'.
```
choco install vcpython27 --yes
```
According to https://github.com/microsoft/azure-pipelines-image-generation/issues/793

**Comment by masayukig** [comment]
Maybe, the next step is moving travis jobs to Azure pipelines?

**Comment by mtreinish** [comment]
I think we should just start by doing windows testing on this. Once we've used it for a little while we can look at fully replacing travis too.

**Comment by masayukig** [comment]
Yeah, I agree. Let's see the stability, performance, etc.
Do you have anything else to do for this pull request?

**Pull request merged**